yal/tests/test_quiz.py

```diff
@@ -17,11 +17,9 @@ def tester():
 async def contentrepo_api_mock(sanic_client):
     Sanic.test_mode = True
     app = Sanic("contentrepo_api_mock")
-    app.requests = []
```

Comment:
Why isn't this one replaced with tstate?

Reply:
Because nothing was using it and the tests passed without it. We can put it back if it's needed for something else, though.

Reply:
I see